### DISCUSSION TITLE
Allow for using `0` for the Normal step in the Exposure time finder

### DISCUSSION THREAD

**Discussion by Pauan** [sn4k3/UVtools#958] [Ideas]
I'm trying to use the "Multiple exposures" feature to calibrate the optimal bottom exposure time for my printer.

However, it does not allow me to enter `0` into the "Normal step" field. It is clamped at a minimum of `0.01`.

![exposure normal step](https://github.com/user-attachments/assets/ad2d9e03-e50c-4386-88b4-e8540c399d24)

This means that it will increment the normal exposure on each test... I don't want that.

I only want to increment the bottom exposure, not the normal exposure.

This can be fixed by simply allowing users to use `0` for the "Normal step".

**Comment by sn4k3** [comment]
Last release fix that.

**Reply by Pauan** [reply]
I have confirmed that it is working in the latest version. Thank you so much!
